man/org.freedesktop.login1.xml

```diff
@@ -1268,6 +1271,11 @@ node /org/freedesktop/login1/session/1 {
       connection. This should help prevent a session from entering an inconsistent state, for example if the
       controller crashes. The only argument <varname>type</varname> is the new session type.</para>
 
+      <para><function>SetDisplay()</function> allows to change the display name of a graphical session. This is useful
+      if the display server is started as part of the session. It can only be called by session's current controller.
+      If <function>TakeControl()</function> has not been called, this method will fail. The only argument
+      <varname>display</varname> is the new display name.</para>
```

Comment:
`display` -> `name` (or at least `type` if you'd prefer the current argument name.)

Reply:
I dont understand 😖